### ISSUE TITLE
Campfire volume slider should support dark theme

### ISSUE THREAD

**Issue by huyenltnguyen** [freeCodeCamp/freeCodeCamp#52730]
### Describe the Issue

The campfire volume slider is hard to see in dark theme.

| Light | Dark |
| --- | --- |
| <img width="788" alt="Screenshot 2023-12-23 at 17 22 58" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/25715018/cac7537a-1621-4827-bf1b-56dbf5b3f081"> | <img width="800" alt="Screenshot 2023-12-23 at 17 20 53" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/25715018/67eeb3d5-c4b3-4c97-a385-e2da196bf021"> |

This doesn't seem to be a regression, but we just haven't supported dark theme. Original implementation: https://github.com/freeCodeCamp/freeCodeCamp/pull/47259/files.

### Affected Page

https://www.freecodecamp.org/settings

### Steps to Reproduce

1. Go to https://www.freecodecamp.org/settings
2. Scroll down and find the Campfire Volume setting

**Comment by bbsmooth** [comment]
Put a white border around it, like we do for the buttons?

**Comment by ahmaxed** [comment]
This is what I see in production.
The interactive element should have hight contrast borders as @bbsmooth suggested.
In addition, flipping the colors for the whole slider on dark mode would be optimal.

<img width="862" alt="Screenshot 2024-01-02 at 9 02 15 AM" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/4591597/99745fbd-cc97-40ab-9126-928e0ed84398">
